SIGS_AND_WGS.md

```diff
@@ -0,0 +1,25 @@
+# SIGs and WGs
+
+We follow the Kubernetes model for [SIGs] and [WGs].
+
+## [SIGs] (Special Interest Groups)
+
+Our [SIGs] define mission and scope via their charter, ownership of code via [OWNERS](https://www.kubernetes.dev/docs/guide/owners/), and define members and roles in community [sigs.yaml].
+
+Examples:
+* [sig-network - charter](./sig-network/charter.md)
+* [sig-network - kubevirt/kubevirt OWNERS_ALIASES](https://github.com/kubevirt/kubevirt/blob/a7e0311d8704663351abd4bc9bbc8511753d2838/OWNERS_ALIASES#L60)
+* [sig-ci - sigs.yaml](https://github.com/kubevirt/community/blob/4f63a79c0ed810aa332cd6716d4986001d28bcd7/sigs.yaml#L119)
```

Comment:
This link seems to be broken.

![image](https://github.com/kubevirt/community/assets/156769580/225d8907-bc51-4194-ac2c-715b1be7e495)